src/webviews/home/protocol.ts

```diff
@@ -189,32 +184,57 @@ export interface GetOverviewBranch {
 		uri: string;
 	};
 }
-export interface GetOverviewBranches {
-	active: GetOverviewBranch[];
-	recent: GetOverviewBranch[];
-	stale: GetOverviewBranch[];
+
+export interface OverviewRepository {
+	name: string;
+	path: string;
+	provider?: {
+		name: string;
+		icon?: string;
+		url?: string;
+	};
 }
 
-export type GetOverviewResponse =
+export interface GetActiveOverviewRequest {
+	[key: string]: unknown;
```

Comment:
These need to be typed

Reply:
We don't actually send anything, I think I just forgot to delete these types.